engine/src/field.h

```diff
@@ -239,7 +240,9 @@ class MCField : public MCControl, public MCMixinObjectHandle<MCField>
 	MCStringRef label;
     MCTextDirection text_direction;
     MCInterfaceFieldCursorMovement cursor_movement;
-    
+    MCInterfaceKeyboardType keyboard_type;
```

Comment:
These should be bit fields :)

Reply:
ah.. woops... had some fun with my patches being a bit overlapping and must have forgotten to put those back in. just a sec!

Reply:
done